projects/gloo/cli/pkg/cmd/install/installer.go

```diff
@@ -188,31 +190,35 @@ func (i *installer) printReleaseManifest(release *release.Release) error {
 }
 
 // The resulting URI can be either a URL or a local file path.
-func getChartUri(chartOverride string, withUi bool, enterprise bool) (string, error) {
-	var helmChartArchiveUri string
-	enterpriseTag, err := version.GetEnterpriseTag(true)
-	if err != nil {
-		return "", err
-	}
-	// Overrides
-	if version.EnterpriseTag != version.UndefinedVersion {
-		enterpriseTag = version.EnterpriseTag
+func getChartUri(chartOverride, releaseVersionOverride string, withUi, enterprise bool) (string, error) {
+
+	if chartOverride != "" && releaseVersionOverride != "" {
+		return "", errors.Errorf("you may only provide one override value, detected two: %s and %s", chartOverride, releaseVersionOverride)
```

Comment:
```suggestion
		return "", errors.Errorf("you may not specify both a chart with -f and a release version with --release-version. Received: %s and %s", chartOverride, releaseVersionOverride)
```
Little wording nit, but I'm not sure that we've really established the idea of an "override" as a noun in users' heads. Better to be explicit here.

Also, can we change this error declaration into a function in an `errors.go` file, and get some test coverage of this path?